cluster/local/integration_tests.sh

```diff
@@ -191,7 +191,7 @@ echo "${INSTALL_YAML}" | "${KUBECTL}" delete -f -
 timeout=60
 current=0
 step=3
-while [[ $(kubectl get providerrevision.pkg.crossplane.io -o name | wc -l) != "0" ]]; do
+while [[ ! $(($(kubectl get providerrevision.pkg.crossplane.io -o name | wc -l))) ]]; do
```

Comment:
If you just want to check the exit code, I believe you can just do 

```bash
while ! kubectl get providerrevision.pkg.crossplane.io -o name | wc -l; do
```

However, that doesn't work here since the exit code is always zero. You would have to do something like

```bash
while [ "$(kubectl get providerrevision.pkg.crossplane.io -o name | wc -l)" != "0" ]; do
```

That should also work with `/bin/sh`.

Reply:
The problem is that the string `"0"` does not work on MacOS X. `wc` prints the count with whitespace. For example `"        0"`. So a simple string compare doesn't work unless you trim the spaces one way or another. The loop keeps going until there are 0 providers running.

It captures stdout and checks if it's 0. The original code compares a string (broken when using MacOS X `wc`) but converting it to an integer will remove the spaces and the truthiness of the resulting integer can be used instead of the string.

Reply:
`$(( "$x" ))` converts `x` to an integer.

It's 100% valid Bash syntax. The script's header says that it's a Bash script and it already used double brackets. The trimming of whitespaces is the goal of my change. The solution works on all systems that have Bash installed.

Reply:
Then I would suggest a K8s native way like
```bash
kubectl wait providerrevision.pkg.crossplane.io --for=delete --all
```

Reply:
Good point. I changed it to use that instead and it works fine.